https://bugs.kde.org/show_bug.cgi?id=462007
https://sourceware.org/git/?p=valgrind.git;a=commit;h=0811a612dd7ce0c02a5dd699b34e660c742df8fe

From 0811a612dd7ce0c02a5dd699b34e660c742df8fe Mon Sep 17 00:00:00 2001
From: Mark Wielaard <mark@klomp.org>
Date: Fri, 18 Nov 2022 20:12:06 +0100
Subject: [PATCH 1/1] Implicit int in none/tests/faultstatus.c

There is a definition in faultstatus.c that is not accepted by
C99 compilers (implicit ints were removed in that language revision).

https://bugs.kde.org/show_bug.cgi?id=462007
--- a/none/tests/faultstatus.c
+++ b/none/tests/faultstatus.c
@@ -190,7 +190,7 @@ int main()
 	return 0;
 }
 
-static volatile s_zero;
+static volatile int s_zero;
 
 static int zero()
 {
